model_analyzer/config/generate/brute_plus_binary_search_run_config_generator.py

```diff
@@ -116,7 +116,11 @@ def _create_brute_run_config_generator(self) -> BruteRunConfigGenerator:
 
     def _can_binary_search_top_results(self) -> bool:
         for model in self._models:
-            if model.parameters()["concurrency"] or model.parameters()["request_rate"]:
+            if (
+                model.parameters()["concurrency"]
+                or model.parameters()["request_rate"]
+                or self._config.is_llm_model()
```

Comment:
What is this if statement checking and will it ever be false?

Reply:
It's determining if we should be a binary search of the concurrency/request-rate space for our top configurations. I'm disabling this for LLM models as it's not relevant.